packages/account/src/Styles/account.scss

```diff
@@ -1579,230 +1579,6 @@ $MIN_HEIGHT_FLOATING: calc(
     }
 }
 
-/** @define connected-apps; weak */
```

Comment:
The page is not responsive enough in desktop view, please have another look and fix the issues.

Reply:
<img width="973" alt="Screenshot 2023-10-13 at 2 07 41 PM" src="https://github.com/binary-com/deriv-app/assets/70938039/fa7e9639-bcc9-4740-9dfe-3a910bed5de8">

Headers look different from the design!

Reply:
In the info message, the alignments are off and not following the design.

Reply:
these are resolved in [this card](https://app.clickup.com/t/20696747/WALL-2141).